src/ctia/bulk/core.clj

```diff
@@ -69,11 +69,15 @@
   "Retrieve many entities of the same type provided their ids and common type"
   [ids entity-type auth-identity]
   (let [read-entity (read-fn entity-type auth-identity {})]
-    (map (fn [id] (try (with-long-id
-                         (read-entity id))
-                       (catch Exception e
-                         (do (log/error (pr-str e))
-                             nil)))) ids)))
+    (map (fn [id]
+           (try
+             (if-let [entity (read-entity id)]
+               (with-long-id entity)
+               (do (log/warnf "bulk/read-entities: Entity not found: %s" id)
```

Comment:
Please don't log this case. It happens for all target entities of Relationships that are not in the same CTIA instance.